src/PregWrapper.php

```diff
@@ -0,0 +1,175 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer;
+
+/**
+ * @author Kuba Werłos <werlos@gmail.com>
```

Comment:
There is missing reasoning why one shall use that class instead of `preg_*` methods directly

Reply:
Added, open for suggestion for better one.